tasks/rh-sign-image/rh-sign-image.yaml

```diff
@@ -68,6 +68,22 @@ spec:
         N=$(params.concurrentLimit)  # The maximum number of images to be processed at once
         count=0
         COMPONENTS_LENGTH=$(jq '.components |length' ${SNAPSHOT_PATH})
+
+        # Add a function to check if the image is already signed
```

Comment:
It doesn't make sense to have a comment in the code saying "add something". That only makes sense in changelog or a commit message. Instead, here it can say "A function to check...".